sentry_sdk/integrations/opentelemetry/integration.py

```diff
@@ -6,10 +6,8 @@
 
 from sentry_sdk.integrations import DidNotEnable, Integration
 from sentry_sdk.integrations.opentelemetry.propagator import SentryPropagator
-from sentry_sdk.integrations.opentelemetry.span_processor import (
-    SentrySpanProcessor,
-)
 from sentry_sdk.opentelemetry.sampler import SentrySampler
+from sentry_sdk.opentelemetry.span_processor import SentrySpanProcessor
```

Comment:
```suggestion
from sentry_sdk.opentelemetry import SentrySampler, SentrySpanProcessor
```

Why not use the re-exports here? Would reduce the number of import statements we need. If we have the re-exports, we should use them; otherwise, we should get rid of the re-exports.

Reply:
let's use them 👍🏻